image/templates/helm/shared/internal/scanner-v4-config-shape.yaml

```diff
@@ -0,0 +1,81 @@
+scannerV4:
+  disable: null # bool
+  indexer:
+    metricsPort: null # int
+    disable: null # bool
+    logLevel: null # string
+    image:
+      registry: null # string
+      name: null # string
+      tag: null # string
+      fullRef: null # string
+    replicas: null # int
+    autoscaling:
+      disable: null # bool
+      minReplicas: null # int
+      maxReplicas: null # int
+    nodeSelector: null # string | dict
+    tolerations: null # [dict]
+    affinity: null # dict
+    resources: null # string | dict
+    serviceTLS:
+      cert: null # string
+      key: null # string
+      generate: null # bool
+  matcher:
+    metricsPort: null # int
```

Comment:
depending on whether indexer/matcher communicate, would be nice to specify the following
```
indexer:
  matcher:
    host: 
    port:
    tls:
      certificates:
      - secretName: ...
```

Reply:
the matcher does have to talk to the indexer (not the other way around, but let's worry about that in a followup PR)